### PR TITLE
mayhem integration

### PR DESCRIPTION
**Pull request by evanrichter**

### CODE REVIEW

File: mayhem/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Build Stage
```
Comment (xansec) [inline mayhem/Dockerfile:1]: Can we add a package stage here?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: mayhem/Dockerfile (hunk 1 above)
Comment (evanrichter) [reply mayhem/Dockerfile:1]: done! there are quite a few shared libraries :sweat_smile:

**Comment by xansec** [comment]
@evanrichter Did you sync with upstream recently? There might have been a breaking change:
https://github.com/mayhemheroes/librsvg/runs/7663035382?check_suite_focus=true#step:5:2073
Please open a new PR when resolved.

**Comment by evanrichter** [comment]
no I haven't will do that now

**Comment by evanrichter** [comment]
see #2

**Pull request merged**